Shakespeare-data_cleaning/Data cleaning.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+def process_paragraphs(input_file, output_paragraphs, output_bin):
+    with open(input_file, 'r', encoding='utf-8') as infile, \
+            open(output_paragraphs, 'w', encoding='utf-8') as out_paragraphs, \
+            open(output_bin, 'w', encoding='utf-8') as out_bin:
```

Comment:
I would stick to uniform naming for file handles, e.g.
input_handler
output_paragraphs_handler
output_bin_handler

(can switch from handler to file, or file_handler)